### PR TITLE
i18n(es): Update `authoring-content`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Updated `authoring-content`

### CODE REVIEW

File: docs/src/content/docs/es/guides/authoring-content.md [hunk 1]
```diff
@@ -7,6 +7,22 @@ Starlight admite la gama completa de la sintaxis [Markdown](https://daringfireba
 
 Por favor, asegúrate de consultar la documentación de [MDX](https://mdxjs.com/docs/what-is-mdx/#markdown) o [Markdoc](https://markdoc.dev/docs/syntax) si estás utilizando esos formatos de archivo, ya que el soporte y el uso de Markdown pueden variar.
 
+## Frontmatter
+
+Puedes personalizar individualmente las páginas en Starlight estableciendo valores en su frontmatter.
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/guides/authoring-content.md:12]:
```suggestion
Puedes personalizar individualmente las páginas en Starlight estableciendo valores en el frontmatter.
```

File: docs/src/content/docs/es/guides/authoring-content.md [hunk 2]
```diff
@@ -7,6 +7,22 @@ Starlight admite la gama completa de la sintaxis [Markdown](https://daringfireba
...
+El frontmatter se establece en la parte superior de tus archivos entre separadores `---`:
+
+```md title="src/content/docs/example.md"
+---
+title: Mi título de página
+---
+
+El contenido de la página sigue el segundo `---`.
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/guides/authoring-content.md:20]:
```suggestion
El contenido de la página sigue luego de los `---`.
```

File: docs/src/content/docs/es/guides/authoring-content.md [hunk 3]
```diff
@@ -7,6 +7,22 @@ Starlight admite la gama completa de la sintaxis [Markdown](https://daringfireba
...
+```
+
+Cada página debe incluir al menos un `title`.
+Ver la [referencia de frontmatter](/es/reference/frontmatter/) para ver todos los campos disponibles y cómo añadir campos personalizados.
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/guides/authoring-content.md:24]:
```suggestion
Vea la [referencia de frontmatter](/es/reference/frontmatter/) para ver todos los campos disponibles y cómo añadir campos personalizados.
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Thanks @dreyfus92, so many PRs to review 💜

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @kevinzunigacuellar 🫡

**Pull request merged**